pandas/core/indexes/base.py

```diff
@@ -2005,15 +2005,35 @@ def __contains__(self, key):
             return False
 
     _index_shared_docs['contains'] = """
-        return a boolean if this key is IN the index
+        Return a boolean if this key is in the index.
 
         Parameters
         ----------
-        key : object
+        key : label
+            The key can be of the same type as the label of :class: `Index`,
+            hence immutable-like and 1-dimensional if it is a tuple.
 
         Returns
         -------
-        boolean
+        bool
+            Result of the key search.
+
+        See Also
+        --------
+        Index.isin : Returns ndarray of boolean dtype if list-like key is in
+            index.
+
+        Examples
+        --------
+        >>> idx = pd.Index([1, 2, (3, 4), 5])
+        >>> idx
+        Index([1, 2, (3, 4), 5], dtype='object')
+        >>> idx.contains((3,4))
+        True
```

Comment:
Small thing, but do you mind moving this example after the other two? I think it helps if we show the simple/easy cases first, and the more complex later.

Also, you there is a missing space after the comma.

When you're done with the changes, it may be helpful to run `./scripts/validate_docstrings.py pandas.Index.contains` to make sure there are not errors (I don't think there are).

And also `flake8 --doctests pandas/core/indexes/base.py`. Note that this will report all doctests problems in all the docstrings of the file. Just look that there is no problem in the lines of this docstring 2008-2036.

Reply:
I have made the said changes.

Validation tests passed with a warning of absence of an extended description which has been suggested to avoid in the review of the original PR. Tested via command `./scripts/validate_docstrings.py pandas.Index.contains`

No doctests problem were reported for lines 2008-2036 by testing via command `flake8 --doctests pandas/core/indexes/base.py`